sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoads.java

```diff
@@ -160,40 +164,41 @@ public void validate(PipelineOptions options) {
   @Override
   public WriteResult expand(PCollection<KV<DestinationT, TableRow>> input) {
     Pipeline p = input.getPipeline();
-    final String stepUuid = BigQueryHelpers.randomUUIDString();
-
-    PCollectionView<String> tempFilePrefix =
-        p.apply("Create", Create.of((Void) null))
-            .apply(
-                "GetTempFilePrefix",
-                ParDo.of(
-                    new DoFn<Void, String>() {
-                      @ProcessElement
-                      public void getTempFilePrefix(ProcessContext c) {
-                        c.output(
-                            resolveTempLocation(
-                                c.getPipelineOptions().getTempLocation(),
-                                "BigQueryWriteTemp",
-                                stepUuid));
-                      }
-                    }))
-            .apply("TempFilePrefixView", View.<String>asSingleton());
 
     // Create a singleton job ID token at execution time. This will be used as the base for all
     // load jobs issued from this instance of the transform.
-    PCollectionView<String> jobIdTokenView =
+    final PCollectionView<String> jobIdTokenView =
```

Comment:
outside scope of this PR: such singleton stuff starts being a recurring pattern, wonder if we should have a `Create.of(SerializableFunction<Void, T>)` overload.